src/d3a/models/area/redis_dispatcher/__init__.py

```diff
@@ -0,0 +1,36 @@
+from abc import ABC, abstractmethod
+
+
+class RedisEventDispatcherBase(ABC):
```

Comment:
Would be nicer to have docstring explaining base class implementation.

Reply:
Adding docstrings now that the implementation has stabilized.

Reply:
Wouldn't it be better to have this class as part of redis package?

Reply:
IMO no, because this class is the base class for the Area event dispatchers. redis package is not specific for the area, and is supposed to be used by any other part of the code. It is anyways under area/redis_dispatcher, so I think its purpose is clear.

Reply:
Makes sense now, thanks for explaining.